### PR TITLE
Add some Inventories and Terraform templates

### PR DESCRIPTION
**Pull request by ukanga**
Changes:

   - Add Terraform reveal web + superset templates
   - Add sample inventories for reveal_web and superset
   - Adds the shell script  to copy the inventories for a new setup.
   - Adds the shell script  to copy a terraform deployment for a new setup.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +84,30 @@ inventories/
...
+│   └── staging
+│       └── group_vars
+│           ├── reveal_web
+│           │   ├── vars.yml
+│           │   └── vault.yml
+│           └── reveal_web_client
+│               └── vars.yml
+```
+
+   **Note**: The hosts file is not part of the file generated. You will need to generate ypurself, look at the `sample-inventories/inventory-a/hosts` for an example.
```
Comment (evansmurithi) [inline README.md]:
```suggestion
   **Note**: The hosts file is not part of the file generated. You will need to generate yourself, look at the `sample-inventories/inventory-a/hosts` for an example.
```

File: README.md [hunk 2]
```diff
@@ -84,6 +84,30 @@ inventories/
...
+```console
+./scripts/new_inventory.sh reveal_web demo production
+```
+
+Generates the following inventory files:
+
+```console
+inventories
+├── demo
+│   └── staging
```
Comment (morrismukiri) [inline README.md]: Should this be `production` to match the command above?

File: README.md (hunk 1 above)
Comment (morrismukiri) [inline README.md]:
```suggestion
   **Note**: The hosts file is not part of the file generated. You will need to generate it yourself, look at the `sample-inventories/inventory-a/hosts` for an example.
```

File: sample-inventories/inventory-a/group_vars/reveal_web/vars.yml [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+        charset: "UTF-8"
+        proxy_pass: http://localhost:3000
+        proxy_http_version: 1.1
+        proxy_set_header:
+          - "Upgrade $http_upgrade"
+          - "Connection 'upgrade'"
+          - "Host $host"
+          - "X-Forwarded-Proto https"
+        proxy_cache_bypass: $http_upgrade
+reveal_nginx_monit_protection: false
```
Comment (morrismukiri) [inline sample-inventories/inventory-a/group_vars/reveal_web/vars.yml:189]: Should the default behavior be to disable Monit?

Comment (ukanga) [reply sample-inventories/inventory-a/group_vars/reveal_web/vars.yml:189]: I tend to build images than run ansible against an existing setup, you normally would enable this on startup hence the default value has been false.

Comment (morrismukiri) [reply sample-inventories/inventory-a/group_vars/reveal_web/vars.yml:189]: cool

File: README.md (hunk 2 above)
Comment (jasonrogena) [inline README.md]:
```suggestion
│   └── production
```

**Review by jasonrogena — APPROVED** [review]

**Pull request merged**